indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -12,7 +12,7 @@ import React from 'react';
 import {Field} from 'react-final-form';
 import {Form, Input} from 'semantic-ui-react';
 
-import {SingleDatePicker} from 'indico/react/components';
+import DatePicker from 'indico/react/components/DatePicker';
```

Comment:
I would add an export to `indico/web/client/js/react/components/index.js` like we do for all the other components and then keep this as:

```
import {DatePicker} from 'indico/react/components';
```

Reply:
Done

Reply:
Check my last comment again ;) (I think you misunderstood it)

Reply:
Ahh, ok.

Reply:
Done 2.0